components/console_simple_init/console_simple_init.h

```diff
@@ -13,15 +13,27 @@
 extern "C" {
 #endif
 
+
+/* This stucture describes the plugin to the rest of the application */
+typedef struct {
+    /* A pointer to the plugin name */
+    const char *name;
+
+    /* A function which the plugin provides to the application. */
+    esp_err_t (*plugin_regd_fn)(void);
+} console_cmd_plugin_desc_t;
```

Comment:
The descriptions here seem to be copy-pasted from IDF example. Could you try to make them more specific to this particular use case (auto-registration of console commands)?

For example, what is this "function which the plugin provides to the application" — is it just about any function or some specific function? I think it is the specific function which performs registration of console commands...